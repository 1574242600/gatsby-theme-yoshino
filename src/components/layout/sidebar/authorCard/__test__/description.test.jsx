import React from "react";
import renderer from "react-test-renderer";
import Description from "../description";

describe("Description 组件", () => {

    it("快照", () => {
        const tree = renderer
            .create(<Description>test</Description>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });

});
