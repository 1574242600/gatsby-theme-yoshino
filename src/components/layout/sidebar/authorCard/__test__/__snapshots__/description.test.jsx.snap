// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description 组件 快照 1`] = `
<span
  className="description"
>
  test
</span>
`;
